### ISSUE TITLE
idea: ability to delete teams

### ISSUE THREAD

**Issue by TreyWW** [TreyWW/MyFinances#447]
### How big is this feature?

Small

### Describe the problem

Add the ability to delete your team/organization

**Comment by glizondo** [comment]
Hi, @TreyWW are you still considering implementing this?

**Comment by TreyWW** [comment]
Hey @glizondo, yep!

**Comment by TreyWW** [comment]
There's a few requirements though. To delete a team there should be no members (e.g. all users must be kicked). That's basically it though really, from what I can think of.

There will need to be a service layer, view and button in the existing template.

**Comment by glizondo** [comment]
Yes, I was about to ask. Do you have any UI design in mind? Also, I might need to ask you a couple of things here and there (on Discord) if that is ok, but I would like to take on this feature

**Comment by Dean-Katz** [comment]
@TreyWW can I take this issue? Not sure if @glizondo is still working on it or not. Looking to work on my first contribution over the next couple of weeks.

**Comment by TreyWW** [comment]
Hey @Dean-Katz,

This hasn't yet been worked on so yeah that'd be great. Let me know if you'd like to discuss anything.

If I miss the message feel free to just ping me, or if you have discord you can join the server or DM me :)

**Comment by Dean-Katz** [comment]
Sounds good. I will let you know! Thank you.

**Comment by Dean-Katz** [comment]
Hey @TreyWW. Just submitted the first go at the PR for this.
